.storybook/grid/gridStyles.ts

```diff
@@ -1,6 +1,99 @@
-import { css } from '@emotion/core';
+import { css } from '@emotion/react';
+import React from 'react';
 
-const barHeight = "48px"
+const barHeight = '48px';
+
+export const nonEmoStyles: Record<string, React.CSSProperties> = {
+    modal: {
+        position: 'absolute',
+        top: barHeight,
+        left: 0,
+        width: '100%',
+        height: `calc(100% - ${barHeight}`,
+    },
+    iframe: {
+        width: '100%',
+        height: '100%',
+    },
+    buttonRow: {
+        display: 'flex',
+        borderBottom: '1px solid rgba(0,0,0,.1)',
+        margin: '0 15px',
+        padding: '8px 0',
+        justifyContent: 'space-between',
+    },
+    activeButton: {
+        border: '0',
+        cursor: 'pointer',
+        position: 'relative',
+        textAlign: 'center',
+        textDecoration: 'none',
+        transition: 'all 150ms ease-out',
+        transform: 'translate3d(0,0,0)',
+        verticalAlign: 'top',
+        whiteSpace: 'nowrap',
+        margin: '0',
+        fontSize: '12px',
+        fontWeight: 'bold',
+        lineHeight: '1',
+        background: '#fafafa',
+        color: '#333333',
+        boxShadow: 'rgba(0,0,0,.1) 0 0 0 1px inset',
+        borderRadius: '4px',
+        padding: '10px 16px',
+        display: 'inline',
+        overflow: 'visible',
+    },
+    inactiveButton: {
+        border: '0',
+        cursor: 'no-drop',
+        position: 'relative',
+        textAlign: 'center',
+        textDecoration: 'none',
+        transition: 'all 150ms ease-out',
+        transform: 'translate3d(0,0,0)',
+        verticalAlign: 'top',
+        whiteSpace: 'nowrap',
+        margin: '0',
+        fontSize: '12px',
+        fontWeight: 'bold',
+        lineHeight: '1',
+        background: '#dddddd',
+        color: '#aaaaaa',
+        boxShadow: 'rgba(0,0,0,.1) 0 0 0 1px inset',
+        borderRadius: '4px',
+        borderBottomStyle: 'none',
+        padding: '10px 16px',
+        display: 'inline',
+        overflow: 'visible',
+    },
+    resultRow: {
+        display: 'flex',
+        borderBottom: '1px solid rgba(0,0,0,.1)',
+        margin: '0 15px',
+        padding: '8px 0',
+    },
+    resultValue: {
+        appearance: 'none',
+        border: '0 none',
+        boxSizing: 'inherit',
+        display: 'block',
+        margin: '0',
+        background: '#ffffff',
+        padding: '6px 10px',
+        fontSize: '13px',
+        position: 'relative',
+        transition: 'all 200ms ease-out',
+        color: '#333333',
+        boxShadow: 'rgba(0, 0, 0, 0.1) 0 0 0 1px inset',
+        borderRadius: '4px',
+        lineHeight: '20px',
+        flex: '1',
+        textAlign: 'left',
+        overflow: 'visible',
+        height: '52px',
+    },
+};
 
 export const styles = {
```

Comment:
Oh I guess these emotion styles can go now we're not using emotion in this context?

Reply:
I went ahead and removed these.